cmd/diff.go

```diff
@@ -34,7 +34,7 @@ func NewDiffCmd(globalCfg *config.GlobalImpl) *cobra.Command {
 	f.StringVar(&diffOptions.Args, "args", "", "pass args to helm diff")
 	f.StringArrayVar(&diffOptions.Set, "set", nil, "additional values to be merged into the command")
 	f.StringArrayVar(&diffOptions.Values, "values", nil, "additional value files to be merged into the command")
-	f.IntVar(&diffOptions.Concurrency, "concurrency", 0, "maximum number of concurrent downloads of release charts")
+	f.IntVar(&diffOptions.Concurrency, "concurrency", 0, "maximum number of concurrent helm processes to run, 0 is unlimited")
```

Comment:
@stek29 why change the help msg?

Reply:
because it's not strictly correct, it's not only concurrent downloads, but actually number of helm processes to run
and to make it consistent with other commands